services/web/src/app/my/timeline/[id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+'use client';
+import { useAtomValue } from 'jotai';
+
+import TimelineSearch from '../component/TimelineSearch/TimelineSearch';
+import { useGetSearchSchedule } from '../hooks/useGetSearchSchedule';
+import { selectDateAtom } from '../stores/store';
+
+export default function TimelineSearchPage() {
+  const selectedDate = useAtomValue(selectDateAtom);
+
+  const startTime = selectedDate.concat(' 00:00:00');
+  const endTime = selectedDate.concat(' 23:59:59');
```

Comment:
```suggestion
  const startTime = `${selectedDate} 00:00:00`;
  const endTime = `${selectedDate} 23:59:59`;
```

```tsx
const startTime = '2023-02-17 00:00:00';
```

(마이너)
요런 형태를 만들고 싶으셨던게 맞을까요? 
selectedData가 string이 타입인 것 같아서 배열에서도 쓰이는 concat보다는 스트링 리터럴이 더 가독성이 좋은 것 같아서 제안드려봅니닷

Reply:
오 좋아요 이렇게 수정해둘게요:)